store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreRecursionIT.java

```diff
@@ -64,15 +63,14 @@
     private Graph proxyToRestServiceFederatedGraph;
 
     @Before
```

Comment:
This seems superseding parent Annotation between methods tests.

```suggestion
    @Override
```

Reply:
Done